src/main/java/org/prebid/cache/repository/aerospike/AerospikeRepositoryImpl.java

```diff
@@ -92,4 +95,12 @@ private WritePolicy writePolicy() {
                 .retryMax(configuration.getMaxRetry())
                 .exponentialBackoffWithJitter(firstBackoff, maxBackoff);
     }
+
+    private <T>Mono<T> handleAerospikeError(Throwable throwable) {
+        if (throwable instanceof AerospikeException) {
+            return Mono.error(new RepositoryException(throwable.toString()));
```

Comment:
It is a good idea to pass not only exception message to `RepositoryException` but original exception as well so that it is not lost if someone wants to log it later.